packages/compass-editor/src/codemirror/query-history-autocompleter.ts

```diff
@@ -92,3 +108,17 @@ function createInfo(query: SavedQuery): {
     },
   };
 }
+
+// scales a number unscaledNum between [minAllowed, maxAllowed]
+export function scaleBetween(
+  unscaledNum: number,
+  minAllowed: number,
+  maxAllowed: number,
+  min: number,
+  max: number
```

Comment:
took me a bit to understand the min and max params, maybe we could name them differently to make things clearer - for example: 
```
newScaleMin,
newScaleMax,
originalScaleMin,
originalScaleMax,
```

Reply:
Sounds good, thanks for the suggestion!